### PR TITLE
docs(docs): update observability architecture

### PR DESCRIPTION
**Pull request by richardtief**

### CODE REVIEW

File: docs/architecture/monitoring-architecture.png
Comment (ibakshay) [inline docs/architecture/monitoring-architecture.png:1]: Question: Do Plugin and PluginDefinition have n-1 relation?

Comment (ibakshay) [inline docs/architecture/monitoring-architecture.png:1]: Suggestion: Shall we name this box? Is it a namespace? If yes, PluginDefinition needs to be outside? As it is a cluster-wide resource? 🤔
image

File: docs/architecture/monitoring.md [hunk 1]
```diff
@@ -1,31 +1,47 @@
...
 ---
 
-This section provides an overview of the Greenhouse monitoring components. 
+The main terminologies used in this document can be found in [core-concepts](https://cloudoperators.github.io/greenhouse/docs/getting-started/core-concepts).
 
-## Monitoring components
+## Introduction to Observability
 
-1) **Metrics**  
+Observability in software and infrastructure has become essential for operation the complexity of modern Clouds. The concept centers on understanding the internal states of systems based on the data they produce, enabling teams to:
```
Comment (ibakshay) [inline docs/architecture/monitoring.md:10]:
```suggestion
Observability in software and infrastructure has become essential for operating the complex modern cloud environments. The concept centers on understanding the internal states of systems based on the data they produce, enabling teams to:
```

File: docs/architecture/monitoring-architecture.png
Comment (richardtief) [reply docs/architecture/monitoring-architecture.png:1]: Look at the arrow. 🙂 _One Plugin has only one PluginDefinition_ 

But I guess other people will also not understand this so I switched the arrow and make it n-1. 👍🏻

Comment (richardtief) [reply docs/architecture/monitoring-architecture.png:1]: True! Will update the drawing

**Review by ibakshay — APPROVED** [review]
LGTM! 🔥

**Review by viennaa — CHANGES_REQUESTED** [review]
Some small refinements for the lower section. Nice drawing, way better than the old one :)

**Review by viennaa — APPROVED** [review]
<3

**Pull request merged**